### ISSUE TITLE
Typo in Extensions class causes collisions with STRIKETHROUGH, WIKILINKS and ANCHORLINKS

### ISSUE THREAD

**Issue by tcreswick** [sirthias/pegdown#161]
There is an issue with the way that `Extensions` are defined. 

Specifically, there's a collision between `STRIKETHROUGH`, `WIKILINKS` and `ANCHORLINKS`.

This is because:

```` java
static final int WIKILINKS = 0x100;
static final int STRIKETHROUGH = 0x200;
static final int ANCHORLINKS = 0x300;
````
It looks like the bug is simply that `ANCHORLINKS` was supposed to be `0x400` and this is a simple typo.

Unfortunately because the check is done like this:

```` java
   public boolean ext(int extension) {
        return (options & extension) > 0;
    }
````
It means that you get the following collisions:

1. If you enable `WIKILINKS`, it also enables `ANCHORLINKS`
1. If you enable `STRIKETHROUGH`, it also enables `ANCHORLINKS`
1. If you enable `ANCHORLINKS` it also enables `WIKILINKS` and `STRIKETHROUGH`

**Comment by tcreswick** [comment]
Have added a pull request with the fix https://github.com/sirthias/pegdown/pulls

**Comment by eskatos** [comment]
Now that this has been merged, what about a new pegdown version?

**Comment by rabbitfr** [comment]
Hi, 

I stumbled upon this issue, could you please update pegdown ?

Thanks

**Comment by jtzikas** [comment]
+1 to a new release

**Comment by kshchepanovskyi** [comment]
+1 to a new release - I also faced with this issue.

**Comment by sirthias** [comment]
Closed by 8af0001a59690614b1f9dd39c1509e76f714e20c.

**Comment by kshchepanovskyi** [comment]
Thanks!
